04-files.md

```diff
@@ -28,12 +28,12 @@ print(glob.glob('data/inflammation*.csv'))
 ~~~
 
 ~~~ {.output}
-['data/inflammation-01.csv', 'data/inflammation-02.csv', 'data/inflammation-03.csv', 'data/inflammation-04.csv', 'data/inflammation-05.csv', 'data/inflammation-06.csv', 'data/inflammation-07.csv', 'data/inflammation-08.csv', 'data/inflammation-09.csv', 'data/inflammation-10.csv', 'data/inflammation-11.csv', 'data/inflammation-12.csv']
+['data/inflammation-05.csv', 'data/inflammation-11.csv', 'data/inflammation-12.csv', 'data/inflammation-08.csv', 'data/inflammation-03.csv', 'data/inflammation-06.csv', 'data/inflammation-09.csv', 'data/inflammation-07.csv', 'data/inflammation-10.csv', 'data/inflammation-02.csv', 'data/inflammation-04.csv', 'data/inflammation-01.csv']
 ~~~
 
 As these examples show,
-`glob.glob`'s result is a list of strings,
-which means we can loop over it
+`glob.glob`'s result is a list of file and directory paths in arbitrary order.
```

Comment:
@abostroem @tbekolay I have added a short description of the `sorted` built-in function.